# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../ccrtp/gcc43.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ccrtp-1.5.0/src/control.cpp.vanilla	2008-08-10 11:52:30.000000000 +0200
+++ ccrtp-1.5.0/src/control.cpp	2008-08-10 11:52:36.000000000 +0200
@@ -44,6 +44,8 @@
 #include "private.h"
 #include <ccrtp/cqueue.h>
 
+#include <stdlib.h>
+
 #ifdef  CCXX_NAMESPACES
 namespace ost {
 #endif
